examples/map/difference_images.py

```diff
@@ -0,0 +1,62 @@
+"""
+===========================
+Plotting a difference image
+===========================
+How to compute and plot a difference image.
+The example uses `sunpy.map.MapSequence` to compute a difference image and then plot it.
+This basic method works for base difference or running difference. Just change whether
+you're subtracting the previous image or the first image in a sequence.
+"""
+###########################################################################
+
+#sphinx_gallery_thumbnail_number = 2
```

Comment:
For some reason this is showing up in the built documentation. Presumably this should be hidden?

Reply:
I noticed that too. I dunno how to make it go away, though it does appear to be functional as well since the thumbnail in the gallery is the second image.

Reply:
[This example from sphinx-gallery](https://sphinx-gallery.github.io/stable/auto_examples/plot_4_choose_thumbnail.html#sphx-glr-auto-examples-plot-4-choose-thumbnail-py) shows how to use the `sphinx_gallery_thumbnail_number` option, but they seem to output the whole code snippet at the end? Maybe the best option is just to stick it somewhere where it doesn't interrupt the narrative text of your example.

Reply:
I've moved it to the very end in an upcoming commit. We'll see if that breaks anything. If it works, then the gallery template (examples/example_template/example_template.py) should be updated -- I was just following that template.